### PR TITLE
Seperate e2e test run from deployment.

### PR DESCRIPTION
**Pull request by bskiba**

### CODE REVIEW

**Comment by bskiba** [comment]
@schylek

File: vertical-pod-autoscaler/hack/run-e2e-tests.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  print_help
+  exit 1
+fi
+
+SUITE=$1
+
+case ${SUITE} in
+  recommender|updater|admission-controller|actuation|full-vpa)
+    export KUBECONFIG=$HOME/.kube/config
+    go test ${SCRIPT_ROOT}/e2e/*go -v -test.timeout=60m  --args --ginkgo.v=true --ginkgo.focus="\[VPA\] \[${SUITE}\]" --report-dir=/workspace/_artifacts --disable-log-dump
```
Comment (schylek) [inline vertical-pod-autoscaler/hack/run-e2e-tests.sh]: This line is copied from the original script, but SCRIPT_ROOT here (vertical-pod-autoscaler/hack/) is different than there  (vertical-pod-autoscaler/).

File: vertical-pod-autoscaler/hack/run-e2e-tests.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+
+if [ $# -eq 0 ]; then
+  print_help
```
Comment (schylek) [inline vertical-pod-autoscaler/hack/run-e2e-tests.sh]: print_help definition missing

File: vertical-pod-autoscaler/hack/run-e2e-tests.sh (hunk 1 above)
Comment (bskiba) [reply vertical-pod-autoscaler/hack/run-e2e-tests.sh]: I set SCRIPT_ROOT in this script also (line 21).

File: vertical-pod-autoscaler/hack/run-e2e-tests.sh (hunk 2 above)
Comment (bskiba) [reply vertical-pod-autoscaler/hack/run-e2e-tests.sh]: Done

File: vertical-pod-autoscaler/hack/run-e2e-tests.sh (hunk 1 above)
Comment (schylek) [reply vertical-pod-autoscaler/hack/run-e2e-tests.sh]: Oh, OK. I missed "/.." part.

**Pull request merged**